### ISSUE TITLE
Dead Module? Can't change color of tags =[

### ISSUE THREAD

**Issue by Belgaron** [atnoslen/TagIt#75]
Title says it all.

**Comment by ValuedEmployee** [comment]
I'd love to know if this module will continue to get support because there's nothing else like it. It somehow miraculously works for the most part in V11, which I'm grateful for cause I really need it for spell lists tagging

**Comment by Belgaron** [comment]
Yes, I might still try to use it. But the color coding was such a nice
feature.
